### ISSUE TITLE
Tag values always are double quoted enclosed

### ISSUE THREAD

**Issue by zeugfr** [poxet/Influx-Capacitor#8]
When I look at data, tag values text includes double quotes at beginning and ending :
```
show TAG VALUES from processor WITH KEY=hostname
```
returns something like: 
```
"MyHost1"
"MyHost2"
```

And for query `select * from processor limit 1`:
```
time	category	counter	hostname	instance	value
2015-09-04T07:54:06Z	""Processor""	""% Processor Time""	""MyHost1""	""0""	7.587343
```
That forces me to make all my queries with text tags enclosed by double quotes and simple quotes :
```
select * from processor where hostname='"MyHost1"'
```

Is it normal ?

Thanks.

PS: This application is great, exactly what i was searching for monthes :+1:

**Comment by poxet** [comment]
When using some characters (like space) quotes are needed, I believe. Also I use another github project that is supposed to take care of this. But as you explain it it seems strange, so I will have a look at it.

**Comment by poxet** [comment]
This is fixed with version 1.0.0.9
